csc/values-dsm1-pvc.yaml

```diff
@@ -0,0 +1,12 @@
+image:
+  repository: lsstts/dsm
+  tag: develop
+  pullPolicy: IfNotPresent
```

Comment:
if you're pulling develop, and that's a tag you keep pushing to, you might want to say pullPolicy: Always

Reply:
I use Always up in the ArgoCD configuration.